pages/news-grid.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: news-grid
+template-title: News
+template-description: Card-based layout that displays recent news articles or case studies
+title: News
+description: Card-based layout that displays recent news articles or case studies
+intro: 18F partners with federal agencies to improve the user experience of government services by helping them build and buy technology. If you're looking to implement a requirement, update a public-facing website, or digitize a process, 18F can work with you to build a product or craft and execute an effective agile acquisition strategy.
+contact-email:
```

Comment:
Remember to enter something here. Otherwise nothing gets pulled into the page and it shows up as an empty link.